### DISCUSSION TITLE
part-ii/2021-5/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#346] [Question comments]
Part II 2021 Algebraic Topology: Paper 2, Section II, 21F 

https://questions.tripos.org/part-ii/2021-5/

**Comment by artie2000** [comment]
(a) is bookwork (attaching a 2-cell).

<details>
<summary>(b)</summary>

Covering maps are injective.

$\pi_1(T^2)\equiv \mathbb{Z}^2$ is infinite and torsion-free, but $\pi_1(\mathbb{R}P^2)\equiv \mathbb{Z}^2$ is finite and torsion.

Therefore neither can be a subgroup of the other.

</details>
<details>
<summary>(c)</summary>

Contract by a homotopy equivalence to $\bigvee_{i=1}^5 S^1$. I get

$\langle a^4,b^2a^2,b^{-2}a^2,ba,ba\rangle\leq F_2.$

</details>

**Reply by Kiolt** [reply]
(c) seems wrong, $b$ is not the image of a loop in the covering space. I got $\langle a^4,ab\rangle$.

**Reply by artie2000** [reply]
yup sorry I had a copying error, should be fixed now

I disagree with your answer; for example, surely $ba$ is in there?

**Reply by Kiolt** [reply]
You're right, bad mistake on my part.

**Reply by asheshbati** [reply]
That homotopy equivalence is nice - also in this case as the covering is normal, the subgroup itself is also normal, so as @Kiolt says, we get a _normal_ subgroup generated by $a^4$, $ab$. Then we can check this is index 4 via a quotient (where we get $\mathbb{Z}/4\mathbb{Z}$. 

We can use normality to actually calculate the generators of the group as well - i.e. we just need to keep adding generators we need via conjugating our set by $a$, $b$.

**Reply by artie2000** [reply]
That's a nice way to do it! How do you know you're done adding generators tho?

**Reply by asheshbati** [reply]
Eventually you get to a point when if you conjugate any of your generators by $a$ or $b$, you'll get a product your generators, and you can conclude.

**Comment by Kiolt** [comment]
<details>
<summary>(a)</summary>

Remove a disc.

</details>

**Comment by ArthurConmy** [comment]
<details>
<summary>Another description?</summary>

By considering the push-forward of the fundamental group of the covering space, the subgroup is the $ab$ strings

$$ a^{i_1} b^{i_2} ... b^{i_{2n}} $$

where $ i_1 + i_3 + ... + i_{2n-1} \equiv i_2 + i_4 + ... + i_{2n} \pmod{4} $

</details>
